media/starboard/starboard_renderer.cc

```diff
@@ -183,16 +183,29 @@ void StarboardRenderer::SetCdm(CdmContext* cdm_context,
     return;
   }
 
-  drm_system_ = cdm_context->GetSbDrmSystem();
+  cdm_context_ = cdm_context;
+  drm_system_ = cdm_context_->GetSbDrmSystem();
   std::move(cdm_attached_cb).Run(true);
   LOG(INFO) << "CDM set successfully.";
+
+  if (state_ != STATE_INIT_PENDING_CDM) {
+    return;
+  }
+
+  DCHECK(init_cb_);
+  state_ = STATE_INITIALIZING;
+  CreatePlayerBridge();
 }
 
 void StarboardRenderer::Flush(base::OnceClosure flush_cb) {
   DCHECK(task_runner_->RunsTasksInCurrentSequence());
   DCHECK(!pending_flush_cb_);
   DCHECK(flush_cb);
 
```

Comment:
Are we confident to not reset `state_` in `Flush()`?

Reply:
Added `STATE_FLUSHED` after initialization or after flush completed.